opendbc/car/extra_cars.py

```diff
@@ -45,7 +52,8 @@ class CAR(Platforms):
       CommunityCarDocs("Acura TLX 2015-17", "Advance Package"),
       CommunityCarDocs("Acura TLX 2018-20", "All"),
       GMSecurityCarDocs("Acura ZDX 2024", "All"),
-      CommunityCarDocs("Honda Accord 2016-17", "Honda Sensing"),
+      HondaSerialSteeringCarDocs("Honda Accord 2016-17", "Honda Sensing"),
+      HondaSerialSteeringCarDocs("Honda Accord Hybrid 2017", "All"),
```

Comment:
You can put these in the car port directly. Look at the Toyota SecOC cars as an example.